applications/GeoMechanicsApplication/CMakeLists.txt

```diff
@@ -0,0 +1,167 @@
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+
+message("**** configuring KratosGeoMechanicsApplication ****")
+
+# PYBIND11
+include(pybind11Tools)
+
+include_directories( ${CMAKE_SOURCE_DIR}/kratos )
+include_directories( ${CMAKE_SOURCE_DIR}/applications/StructuralMechanicsApplication )
+include_directories( ${CMAKE_SOURCE_DIR}/applications/GeoMechanicsApplication )
+
```

Comment:
I would strongly suggest to use `kratos_add_dependency` for this, e.g. like [here](https://github.com/KratosMultiphysics/Kratos/blob/master/applications/CableNetApplication/CMakeLists.txt#L8)